packages/near-fast-auth-signer/src/components/CreateAccount/CreateAccount.tsx

```diff
@@ -4,6 +4,7 @@ import { useCallback, useEffect } from 'react';
 import { useForm } from 'react-hook-form';
 import { Link, useNavigate, useSearchParams } from 'react-router-dom';
 import styled from 'styled-components';
+import isEmail from 'validator/lib/isEmail';
```

Comment:
We don't need to install an extra package, yup already haves a email validation. 

You can do 

```
yup
    .string()
    .required('Email address is required')
    .email('Invalid email')
```

Reply:
your suggestion doesn't block this type of emails:
`asodifnaiosdn@adiofnasoid.1234`

this PR's change does block:
<img width="377" alt="image" src="https://github.com/near/fast-auth-signer/assets/6027014/a47505af-f89d-4983-9196-ded3dbaac16e">